openfisca_france/model/prestations/enseignement_superieur/sante_psy_etudiant.py

```diff
@@ -0,0 +1,17 @@
+from openfisca_france.model.base import *
+
+
+class seances_sante_psy_etudiant(Variable):
+    value_type = int
+    entity = Individu
+    label = "Nombre de séances prises en charge par l'Aide Santé Psy Étudiant"
+    definition_period = MONTH
+    reference = "https://www.service-public.fr/particuliers/actualites/A14726"
+
+    def formula_2021_03_10(individu, period, parameters):
+        '''
+        Le nombre maximum de séances est donné sans prendre en compte les passages obligatoires intermédiaires chez un médecin.
+        '''
+        eligibilite = individu('etudiant', period)
```

Comment:
La formule actuelle de la variable [`etudiant`](https://fr.openfisca.org/legislation/etudiant) n'est pas tout à fait juste car les jeunes du secondaire (lycée) sont considérés étudiants. Pourras-tu faire une itération d'amélioration de cette formule et regarder les implications pour les formules faisant déjà référence à `etudiant` ?